internal/model/client.go

```diff
@@ -0,0 +1,151 @@
+package model
```

Comment:
Not sure if this is the case, but if we would like to use client and model in different services we probably want to move it out of `internal` directory as packages from it cannot be imported outside of the project itself.

Reply:
Good call!

Reply:
Done :)